cmd/xl-v1-object.go

```diff
@@ -198,8 +198,14 @@ func (xl xlObjects) GetObject(bucket, object string, startOffset int64, length i
 		return traceError(InvalidRange{startOffset, length, xlMeta.Stat.Size})
 	}
 
+	// Calculate endOffset according to length
+	endOffset := startOffset
```

Comment:
we can maybe call this lastByteOffset so that `-1` that we do is self explanatory, or add a comment on why `-1` is done.